Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -485,56 +521,62 @@ private void SetLoadJiraPackageButtonContent()
+            }
+            GingerCoreNET.ALMLib.ALMIntegration.eALMType almType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.QC;
             RadioButton rBtn = sender as RadioButton;
             if ((bool)rBtn.IsChecked)
             {
                 switch (rBtn.Name)
                 {
                     case "QCRadioButton":
-                        if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType != ALMIntegration.eALMType.QC)
-                        {
-                             WorkSpace.Instance.Solution.AlmType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.QC;
-                            ClearALMConfigs();
-                        }
+                            almType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.QC;
                         break;
                     case "RQMRadioButton":
-                        if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType != ALMIntegration.eALMType.RQM)
-                        {
-                            WorkSpace.Instance.Solution.AlmType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.RQM;
-                            ClearALMConfigs();
-                            //ALMIntegration.Instance.UpdateALMType(ALMIntegration.eALMType.RQM);
-                            ALMIntegration.Instance.SetALMCoreConfigurations(); //Because RQM need to update the server field from existing package
+                            almType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.RQM;
+                            ALMIntegration.Instance.SetALMCoreConfigurations(almType); //Because RQM need to update the server field from existing package
                             SetLoadPackageButtonContent();
-                        }
                         break;
                     case "RallyRadioButton":
-                        if ((ALMIntegration.eALMType)WorkSpace.Instance.Solution.AlmType != ALMIntegration.eALMType.RALLY)
-                        {
-                            WorkSpace.Instance.Solution.AlmType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.RALLY;
-                            ClearALMConfigs();
-                        }
+                            almType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.RALLY;
                         break;
                     case "JiraRadioButton":
-                        if ((ALMIntegration.eALMType)WorkSpace.Instance.Solution.AlmType != ALMIntegration.eALMType.Jira)
-                        {
-                            WorkSpace.Instance.Solution.AlmType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.Jira;
-                            ClearALMConfigs();
-                        }
+                            almType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.Jira;
+                        break;
+                    case "qTestRadioButton":
+                            almType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.Qtest;
+                        break;
+                    default:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)